### PR TITLE
InterpolatedCamera: use internal_process

### PR DESCRIPTION
**Pull request by ghost**
As of 3.0, built in nodes should use `internal_process` instead of `process` and leave `process` to be overridden by script.

```
[16:26:20] <mhilbrunner> What does Node.set_process_internal() do? Seems it sets a boolean and the node is added to a group on the SceneTree, but what does it do? Why? When would I use this? :)
[16:26:27] <mhilbrunner> Asking because working on the docs currently
[16:48:08] <Akien> mhilbrunner: It enables internal processing (NOTIFICATION_INTERNAL_PROCESS), which is the same as (idle) processing (NOTIFICATION_PROCESS), but used internally by the node for the node's own API (while NOTIFICATION_PROCESS is meant to be used by scripts extending the node behaviour)
[16:48:58] <Akien> mhilbrunner: same for physics processing
[16:49:31] <Akien> mhilbrunner: it's allows nodes to continue behaving properly even if the user disables _process or _fixed_process
[16:50:21] <mhilbrunner> Akien: got it, thanks.
[16:51:07] <Noshyaar[m]> shouldn't interpolatecamera use internal process then?
[16:51:13] <Akien> mhilbrunner: it's new in 3.0 (previously all used the normal _process and _fixed_process, hence issues), so some nodes may not be fully ported to the internal system
[16:51:15] <Akien> Noshyaar: probably, see ^
[16:51:41] <Noshyaar[m]> PR incoming :P
```

@hpvb sorry for the confusion. :P I thought I could attend the merge meeting and explain it there. Unfortunately it was too late at night.

### CODE REVIEW

File: scene/3d/interpolated_camera.cpp [hunk 1]
```diff
@@ -41,7 +41,7 @@ void InterpolatedCamera::_notification(int p_what) {
 				set_physics_process(false);
```
Comment (akien-mga) [inline scene/3d/interpolated_camera.cpp]: Do you know what this is used for? Seems weird in a class that uses (internal) process to disabled physics process.

Comment (akien-mga) [reply scene/3d/interpolated_camera.cpp]: `git blame` says it's been there forever, with no reason that I can think of. I'd say just kill it as part of this commit too.

Comment (Noshyaar) [reply scene/3d/interpolated_camera.cpp]: According to line 112-113 I think it's to disable it if it's running in the editor, so I guess this should be `set_process_internal(false)`

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**